src/cli/onefuzz/cred_wrapper.py

```diff
@@ -0,0 +1,66 @@
+# Wrap credentials from azure-identity to be compatible with SDK that needs msrestazure or azure.common.credentials
+# Need msrest >= 0.6.0
+# See also https://pypi.org/project/azure-identity/
+
+from typing import Any
+
+from azure.core.pipeline import PipelineContext, PipelineRequest
+from azure.core.pipeline.policies import BearerTokenCredentialPolicy
+from azure.core.pipeline.transport import HttpRequest
+from azure.identity import DefaultAzureCredential
+from msrest.authentication import BasicTokenAuthentication
+
+
+class CredentialWrapper(BasicTokenAuthentication):
+    def __init__(
+        self,
+        credential: Any = None,
```

Comment:
Can we restrict these args to be something other than `Any`? It can allow for truly confusing errors, even just within the function body.